Cargo.toml

```diff
@@ -21,6 +21,7 @@ subtle = { version = "^2.2.2", default-features = false }
 zeroize = { version = "1.1.0", default-features = false }
 getrandom = { version = "0.2.0", optional = true }
 ct-codecs = { version = "1.1.1", optional = true }
+serde = { version = "1.0.124", optional = true }
```

Comment:
There seems to be missing an actual `serde` feature-definition in the `[features]` section of this file.

Reply:
I could be misunderstanding this, but I don't think you need to explicitly write down a feature name. Optional dependencies automatically generate features with the same label. Check out [this section of TRPL](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies). Thoughts?

Reply:
You are completely right. I didn't know about this. In fact, given that:
> "Note: A feature in the [feature] table cannot use the same name as a dependency."


this seems to be just right.